### PR TITLE
Rewrite docs on docusaurus

### PR DESCRIPTION
**Pull request by sventime**

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        },
+        {
+          href: 'https://github.com/Supercolony-net/openbrush-contracts',
+          className: 'header-github-link',
+          position: 'right'
+        }
+      ]
+    },
+    footer: {
+      // links: [
```
Comment (xgreenx) [inline docs/docusaurus.config.js:47]: Can we remove that?

File: docs/docs/smart-contracts/timelock-controller.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+---
+sidebar_position: 7
+title: Timelock Controller
+---
 
 This example shows how you can reuse the implementation of
 [timelock-controller](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/governance/timelock-controller).
 
-## Steps
+## Step 1: Include dependencies
```
Comment (xgreenx) [inline docs/docs/smart-contracts/timelock-controller.md:9]: We have the same readme in the `example` folder. Could you update it in the same way? Or maybe you can simplify the readme in examples and create a link on readme from the `docs` folder?)

Comment (sventime) [reply docs/docs/smart-contracts/timelock-controller.md:9]: Done

**Review by xgreenx — APPROVED** [review]
Good! Could you make please squash commit?

**Pull request merged**